README.md

```diff
@@ -143,7 +141,7 @@ a 32-byte hex string (`0x` followed by 64 hexadecimal digits) that denotes the b
     }
     ```
 
--   [`/tx/` submit a signed transaction.](src/controllers/transaction/TransactionSubmitController.ts) Expects a string with hex-encoded transaction in a JSON POST
+-   [`/transaction` submit a signed transaction.](src/controllers/transaction/TransactionSubmitController.ts) Expects a string with hex-encoded transaction in a JSON POST
```

Comment:
Need to update the example below with `/tx` endpoint. I'm fine with deleting the old endpoints from the README, but maybe for now we should add a "Replaces X" note so that people can still ctrl+F for the endpoints they know.